BraveWallet/Crypto/Stores/UserAssetsStore.swift

```diff
@@ -155,7 +161,7 @@ public class UserAssetsStore: ObservableObject {
         withTimeInterval: 0.25, repeats: false,
         block: { [weak self] _ in
           guard let self = self else { return }
-          self.rpcService.network { network in
+          self.rpcService.network(.eth) { network in
```

Comment:
Should we fetch selected coin from Wallet Service here?

Reply:
based on the logic seems like we only want to check the token info for mainnet. so i just used .eth coin type directly.